cmd/podman/utils/utils.go

```diff
@@ -99,3 +102,31 @@ func PrintNetworkPruneResults(networkPruneReport []*entities.NetworkPruneReport,
 	}
 	return errs.PrintErrors()
 }
+
+func IsCheckpointImage(ctx context.Context, namesOrIDs []string) (bool, error) {
```

Comment:
Can you add a comment explaining what the function does and the semantics of the return values?

Reply:
Thanks, I have added a comment here.